integration-tests/src/test/scala/za/co/absa/spline/KafkaSinkSpec.scala

```diff
@@ -76,7 +76,7 @@ class KafkaSinkSpec
 
           (plan2, _) <- captor.lineageOf(
             reader
-              .option("subscribe", s"$topicName,anotherTopic")
+              .option("subscribe", s"$topicName")
```

Comment:
So this one I found a bit odd - EmbeddedKafka was complaining about "anotherTopic" here not existing on this version specifically so I'm guessing Spark is doing something different here trying to read messages from that topic that it didn't before. 

This test is not looking for different behavior based on multiple topic names, but is this acceptable?

Reply:
Good call this fixed it - I put the other topic back in, renamed the test a bit and added a sensible assertion. Thoughts?

Reply:
Looks great, thanks